assets/bundles/bundle.properties

```diff
@@ -2,90 +2,92 @@
 block.cyber-io-virus.name=Virus
-block.cyber-io-virus.description=An infinite spreading virus.
+block.cyber-io-virus.description=An infinitely spreading virus.
 block.cyber-io-land-projector.name=Land Projector
-block.cyber-io-land-projector.description=Projects the holographic land which can support you to build block on it, powered by Plum Star Inc.
+block.cyber-io-land-projector.description=Projects holographic land which can support you to build blocks on it, powered by Plum Star Inc.
 block.cyber-io-holo-floor.name=Holofloor
 block.cyber-io-holo-floor.description=Holographic floor.
 block.cyber-io-holo-wall.name=Holo Wall
-block.cyber-io-holo-wall.description=Holographic wall, powered by Plum Star Inc. It will regenerate health automatically over time.
+block.cyber-io-holo-wall.description=Protects structures from enemy projectiles. Regenerates over time. Powered by Plum Star inc.
 block.cyber-io-holo-wall-large.name=Large Holo Wall
-block.cyber-io-holo-wall-large.description=The large holographic wall, powered by Plum Star Inc. It will regenerate health automatically over time.
+block.cyber-io-holo-wall-large.description=Protects structures from enemy projectiles. Regenerates over time. Powered by Plum Star inc.
 block.cyber-io-anti-virus.name=Antivirus
-block.cyber-io-anti-virus.description=If you cannot play this game, try to close the antivirus.
+block.cyber-io-anti-virus.description=If you cannot play this game, try to close the antivirus. Destroys projectiles and virus blocks in range.
 block.cyber-io-underdrive-projector.name=Underdrive Projector
 block.cyber-io-underdrive-projector.stats.power-gen={0} when it absorbs {1} blocks
-block.cyber-io-underdrive-projector.description=Absorbing the time of surrounding machines.
+block.cyber-io-underdrive-projector.description=Absorbs the time of surrounding machines. Generates power.
 block.cyber-io-hyper-overdrive-sphere.name=Hyper Overdrive Sphere
-block.cyber-io-hyper-overdrive-sphere.description=LIGHT SPEED!!!
+block.cyber-io-hyper-overdrive-sphere.description=LIGHT SPEED!!
 block.cyber-io-deleter.name=Deleter
 block.cyber-io-deleter.description=Radiates holographic waves which has wave-particle duality.\nWave properties: Hit AOE.\nParticle properties: Execute enemies.
 block.cyber-io-deleter.stats.bullet.damage=[stat]{0} + {1}% lost HP [lightgray] damage
 block.cyber-io-deleter.stats.bullet.execution=Execute HP <= [stat]{0}%[lightgray]
 block.cyber-io-TMTRAINER.name=TMTRAINER
 block.cyber-io-TMTRAINER.description=TMTRAINER
+block.cyber-io-TMTRAINER.details=TMTRAINER
 block.cyber-io-prism.name=Prism
-block.cyber-io-prism.description=PRISM can copy your bullets because it is for your security.
+block.cyber-io-prism.description=Clones projectiles the prisms come into contact.
 block.cyber-io-prism-obelisk.name=Obelisk
-block.cyber-io-prism-obelisk.description=It's watching you.
+block.cyber-io-prism-obelisk.description=It's watching you. Boosts Prisms it's touching.
```

Comment:
There is a tip, `Link with a Prism side by side`, when playing is placing it.

Reply:
i know, someone might not get it and disregard it as "useless" and a "waste of time"